portal/src/main/webapp/config_service.jsp

```diff
@@ -45,6 +45,7 @@
             "oncoprint.hide_vus.default",
             "mycancergenome.show",
             "oncokb.public_api.url",
+            "oncokb.merge_icons",
```

Comment:
maybe `merge_icons_by_default`? If we ever would want to give the option to not split the icons at all thru config then this is more explicit